Python/pywarpx/_libwarpx.py

```diff
@@ -74,8 +74,13 @@ def _get_package_root():
 
 try:
     libwarpx = ctypes.CDLL(os.path.join(_get_package_root(), libname))
-except OSError:
-    raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname)
+except OSError as e:
+    value = e.args[0]
+    if any(so in value for so in ["libwarpx.2d.so", "libwarpx.3d.so"]):
+        raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname)
```

Comment:
```suggestion
    if f'{libname}: cannot open shared object file: No such file or directory' in value:
        raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname) from e
```